webpack/plugins.js

```diff
@@ -1,12 +1,12 @@
-var config = require('config');
-var fs = require('fs');
-var ExtractTextPlugin = require('extract-text-webpack-plugin');
-var HtmlWebpackPlugin = require('html-webpack-plugin');
-var webpack = require('webpack');
+var config = require('config'); // eslint-disable-line no-var
```

Comment:
No idea, I seen old JS here, I thought that webpack needed it.
But if it works with ES6, yep, let's use it !